### PR TITLE
Transfer groundControl from staff to admin route

### PR DESCRIPTION
**Pull request by josh1248**
### Description
Addresses #3031 by re-routing ground control from a staff to an admin route.
P.S.: still figuring out the backend. Will further changes be required?

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update
- [ ] Code quality improvements

### How to test

- Enter the test environment as an admin, and see that you are able to enter ground control normally.
- Copy the ground control route (should be '/groundcontrol')
- Switch the test environment to be a staff
- Check that the ground control button is now missing.
- Paste the ground control route. A 404 route should be expected.

- Go to `src/commons/navigationBar/AcademyNavigationBar` and change the ground control route into a staff route.
- Log in as staff. Manually append '/groundcontrol' to the URL to get to the ground control screen.
- Attempt to delete an assessment, or mass publish an assessment through the ground control panel.
- A 403 Forbidden error should show up.
- Other functions, such as unpublishing or publishing of a single submission in the gradings table, should still be possible.

### Checklist

- [X] I have tested this code - I have manually tested the website to check if I broke any frontend-backend links.
- [ ] I have updated the documentation

### CODE REVIEW

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Comment by RichDom2185** [comment]
Waiting for the BE PR before merging.

File: src/commons/sagas/RequestsSaga.ts [hunk 1]
```diff
@@ -250,13 +250,13 @@ export const getAchievements = async (tokens: Tokens): Promise<AchievementItem[]
 };
 
 /**
- * GET /courses/{courseId}/admin/users/{studentCourseRegId}/goals
+ * GET /courses/{courseId}/staff/users/{studentCourseRegId}/goals
```
Comment (RichDom2185) [inline src/commons/sagas/RequestsSaga.ts:253]: Let's not do this. `/admin` doesn't refer to the role; it refers to (course) administration.

It's a sub-resources belonging to the REST resource which is the course itself.

The role check is transparent (i.e. unknown to) the frontend; you shouldn't be able to tell based on the endpoint what are the checks.

**Review by RichDom2185 — CHANGES_REQUESTED** [review]

**Comment by josh1248** [comment]
@RichDom2185 backend PR is ready now. I have replaced all non-dangerous routes to use a "/staff" instead of "/admin" URL to connect to the backend. I have also changed the routes in the backend directly. All seems good, but I am not sure if I have advertently broken some frontend-backend link. Any automated way to test it?

**Comment by RichDom2185** [comment]
> Any automated way to test it?

@josh1248 not really, I think best to just not touch it. Refer to my comments above. 

Basically your endpoints paths should be related to your resource. 

E.g. GET /users/{id}/name

or something like that

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**